src/simulations/local_elec.py

```diff
@@ -53,8 +55,9 @@ def __init__(self, prm):
         self.labels_day_trans = prm['syst']['labels_day_trans']
         self.n_homes = prm['syst']['n_homes']
         self.homes = range(self.n_homes)
+        self.record = Record
```

Comment:
no do not create new blank record objects! we have one overall record object where we record everything